script.js

```diff
@@ -1,11 +1,16 @@
 document.addEventListener("DOMContentLoaded", function () {
+	/*Magic Number*/

```

Comment:
@PrzemekBartecki you don't need to call those values here "magic numbers" :P they are constants and that's all. By the way, from what I know JavaScript's naming convention for constants is to write them upper case, e.g. ANIMATION_DELAY = 1;

Reply:
In JS, let | const | var have the same rules. The convention you are writing about it can by useing in other type language

Reply:
@PrzemekBartecki good to know! I didn't know that JavaScript doesn't have real constant values known from other languages  (const doesn’t ensure that the variable is immutable) and after I have checked one more time I found what you suggested - that UPPER_CASE naming convension doesn't apply here - e.g. AirBnb style guide:
https://github.com/airbnb/javascript/#naming--uppercase